02/PrintRecords.cpp

```diff
@@ -7,24 +7,95 @@ using namespace std;
 
 namespace samples
 {
+	void InputRecords(Record* info)
+	{
+		cout << "First name: ";
+		cin >> info->FirstName;
+		
+		cout << "Last name: ";
+		cin >> info->LastName;
+		
+		cout << "Student ID: ";
+		cin >> info->StudentID;
+
+		cout << "Score:  ";
+		cin >> info->Score;
+	}
+
 	void ManageRecordsExample()
 	{
-		Record info;
+		Record studentInfos[5];
 		ofstream fileStream("studentRecords.dat", ios_base::out);
 
+		int existingStudents = 1;
+
+		studentInfos[0].FirstName = "Pope";
+		studentInfos[0].LastName = "Kim";
+		studentInfos[0].StudentID = "A12345678";
+		studentInfos[0].Score = 80;
+
+		PrintRecords(&studentInfos[0], &fileStream, existingStudents);
+
 		while (true)
 		{
-			cout << "First name: ", cin >> info.FirstName;
-			cout << "Last name: ", cin >> info.LastName;
-			cout << "Student ID: ", cin >> info.StudentID;
-			cout << "Score: ", cin >> info.Score;
+			char input;
+
+			cout << "a: adds another student info. " << "b : Modifies existing info. " << endl;
+			cin >> input;
```

Comment:
if we read only one character.. we will have to clear eof characters or something?